src/plugins/bar-graph/bar-graph-content.ts

```diff
@@ -25,10 +25,9 @@ export const BarGraphContentModel = TileContentModel
     // that we can tell when it changes.
     dataSetId: types.maybe(types.string),
     primaryAttribute: types.maybe(types.string),
-    primaryAttributeColor: types.optional(types.number, 0),
     secondaryAttribute: types.maybe(types.string),
-    // Map of secondary attribute keys to color indices. Each secondary attribute has its own map.
-    secondaryAttributeColorMap: types.optional(types.map(types.map(types.number)), {}),
+    // Map of attribute keys to color indices. Each attribute has its own map.
+    attributeColorMap: types.optional(types.map(types.map(types.number)), {}),
```

Comment:
We should either (a) make sure project folks are aware that any existing bar graphs with manually-assigned secondary colors will lose them, or (b) provide a snapshot preprocessor that moves the existing secondaryAttributeColorMap to attributeColorMap.

Reply:
Good point. I've added a `preProcessSnapshot` to the model to handle that migration.